wpimath/src/main/java/edu/wpi/first/math/geometry/AllianceFlipper.java

```diff
@@ -0,0 +1,169 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package edu.wpi.first.math.geometry;
+
+import java.util.HashMap;
+
+/**
+ * A utility to standardize flipping of coordinate data based on the current alliance across
+ * different years.
+ *
+ * <p>If every vendor used this, the user would be able to specify the year and no matter the year
+ * the vendor's code is from, the user would be able to flip as expected.
+ */
+public class AllianceFlipper {
```

Comment:
```suggestion
public class AllianceSymmetry {
```

Reply:
I think the name 'AllianceFlipper' gives the idea that you're flipping the alliance color, rather than coordinates elements on the field

Reply:
I agree, ill update it asap